### ISSUE TITLE
Problems when importing lots of photos

### ISSUE THREAD

**Issue by pandelis** [codez/ImportPhotoFolders#27]
I'm importing 20 000 photos from Picasa to Apple Photo using the ImportPhotoFolders. I have 21 big folders each corresponding to 1 year. After 2 or 3 folders imported, I began to have  "Cannot Import Item unrecognizable file format or invalid data" error for all the files yet in the different folders. 
It took me a lot of time to find out that I just needed to restart Apple Photos and then, everything is ok.
It may also be worth noting that you need not to have spaces nor parentheses in your files/folders names and only movies recognizable with QuickTime for the script to work.

**Comment by jproudfo** [comment]
Yeah, I've run into the same problem.  There seems to be a limit on how many images Apple Photos will accept.  After it reaches the limit, it just errors out.  Seems to be a common issue in the Apple Support communities, as well.  It may be useful for the script to restart Apple Photos each time it imports a number of photos (200?).

**Comment by clbell123** [comment]
> Yeah, I've run into the same problem. There seems to be a limit on how many images Apple Photos will accept. After it reaches the limit, it just errors out. Seems to be a common issue in the Apple Support communities, as well. It may be useful for the script to restart Apple Photos each time it imports a number of photos (200?).

I went the easy route and just restart after every folder import.  It's working fairly well.  The only issue now is a corrupt image handling needs to be added.

**Comment by the-lazy-fox** [comment]
> > Yeah, I've run into the same problem. There seems to be a limit on how many images Apple Photos will accept. After it reaches the limit, it just errors out. Seems to be a common issue in the Apple Support communities, as well. It may be useful for the script to restart Apple Photos each time it imports a number of photos (200?).
> 
> I went the easy route and just restart after every folder import. It's working fairly well. The only issue now is a corrupt image handling needs to be added.

Hello there,
I’m facing the same issue and restarting doesn’t seem to be an options for me. Everytime I launch the script, it creates a new folder in the library, as a duplicate, with the exact same name and starts from scratch.
Is there a way to instruct the script to “update” the existing folder without reprocessing what it already imported?
Thank you for the guidance
